Makefile

```diff
@@ -31,7 +31,7 @@ APP_LOAD_PARAMS += --path "1517992542'/1101353413'"
 APPVERSION_M=1
 APPVERSION_N=9
 APPVERSION_P=14
-APPVERSION=$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)
+APPVERSION=$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)-xdc
 APP_LOAD_FLAGS= --appFlags 0x240 --dep Ethereum:$(APPVERSION)
```

Comment:
Remove this before merging in `app-ethereum` repository.

Reply:
The modified version label has been removed, Please check now.